### PR TITLE
LPS-95967 company deletion tests

### PR DESCRIPTION
**Pull request by gustavolimav**
Resending https://github.com/liferay-search/liferay-portal/pull/1145#issuecomment-1883642501

https://liferay.atlassian.net/browse/LPS-195967

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
@gustavolimav, just reminding you I wanted a little clarity on the code.

**Comment by gustavolimav** [comment]
@joshuacords we can discuss it today

**Comment by gustavolimav** [comment]
ci:test:sf

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:relevant

File: modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/service/test/SXPBlueprintLocalServiceTest.java [hunk 1]
```diff
@@ -63,8 +80,9 @@ public void testAddSXPBlueprint() throws Exception {
 
 		// Duplicate external reference code in a different company
 
-		User user = UserTestUtil.addCompanyAdminUser(
-			CompanyTestUtil.addCompany());
+		Company company = CompanyTestUtil.addCompany(true);
```
[the remark this answers is not in the document]
Comment (gustavolimav) [reply modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/service/test/SXPBlueprintLocalServiceTest.java]: Hey Joshua, 

Initialize the company was really not necessary.

I used `UserTestUtil.getAdminUser()` as you recommend, thx for the idea!

**Comment by joshuacords** [comment]
Changes look good to me @gustavolimav, feel free to forward once you've analyzed the tests.

**Review by joshuacords — APPROVED** [review]

**Comment by gustavolimav** [comment]
ci:forward:force

**Pull request closed without merging**